docs/tutorials/cloud-init.md

```diff
@@ -0,0 +1,195 @@
+...................
+status: done
+```
+
+
+## Verify that it worked
+
+Restart the distro just to make sure the changes in `/etc/wsl.conf` made by cloud-init will take effect, as listed
+below:
+
+```powershell
+> wsl -t Ubuntu-Preview
+The operation completed successfully.
+> ubuntupreview
+To run a command as administrator (user "root"), use "sudo <command>".
+See "man sudo_root" for details.
+
+Welcome to Ubuntu Noble Numbat (development branch) (GNU/Linux 5.15.137.3-microsoft-standard-WSL2 x86_64)
+
+ * Documentation:  https://help.ubuntu.com
+ * Management:     https://landscape.canonical.com
+ * Support:        https://ubuntu.com/pro
+
+
+This message is shown once a day. To disable it please create the
+/home/cn/.hushlogin file.
+jdoe@mib:~$
+```
+
+Once logged in the new distro instance's shell, verify that:
+
+1. The default user matches what was configured in the user data file (in our case `jdoe`).
+
+```sh
+jdoe@mib:~$ whoami
+jdoe
+```
+
+2. The supplied cloud-config user data was approved by cloud-init validation.
+
+```sh
+jdoe@mib:~$ sudo cloud-init schema --system
+Valid schema user-data
+```
+
+3. The locale is set
+
+```sh
+jdoe@mib:~$ locale
+LANG=pt_BR
+LANGUAGE=
+LC_CTYPE="pt_BR"
+LC_NUMERIC="pt_BR"
+LC_TIME="pt_BR"
+LC_COLLATE="pt_BR"
+LC_MONETARY="pt_BR"
+LC_MESSAGES="pt_BR"
+LC_PAPER="pt_BR"
+LC_NAME="pt_BR"
+LC_ADDRESS="pt_BR"
+LC_TELEPHONE="pt_BR"
+LC_MEASUREMENT="pt_BR"
+LC_IDENTIFICATION="pt_BR"
+LC_ALL=
+
+```
+
+4. The packages were installed and the commands they provide are available
+
+```sh
+jdoe@mib:~$ apt list --installed | egrep 'ginac|octave'
+
+WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
+
+ginac-tools/noble,now 1.8.7-1 amd64 [installed]
+libginac11/noble,now 1.8.7-1 amd64 [installed,automatic]
+octave-common/noble,now 8.4.0-1 all [installed,automatic]
+octave-doc/noble,now 8.4.0-1 all [installed,automatic]
+octave/noble,now 8.4.0-1 amd64 [installed]
+```
+
+5. Verify that the commands requested were also run. In this case we set up `vcpkg` from git, as recommended by its
+   documentation (there is no deb or snap available for that program).
+
+```sh
+jdoe@mib:~$ /opt/vcpkg/vcpkg version
+vcpkg package management program version 2024-01-11-710a3116bbd615864eef5f9010af178034cb9b44
+
+See LICENSE.txt for license information.
+```
+
+## Enjoy!
+
+That’s all folks! In this tutorial, we’ve shown you how to use cloud-init to automatically set up Ubuntu on WSL 2 with minimal touch.
+
+This workflow will guarantee a solid foundation for your next Ubuntu WSL project.
+
+We hope you enjoy using Ubuntu inside WSL!
+
+### Further Reading
+
+To know more about cloud-init support for WSL check out the [WSL data source reference in cloud-init documentation](https://cloudinit.readthedocs.io/en/latest/reference/datasources/wsl.html).
+To learn more about cloud-init in general visit [cloud-init official documentation](https://cloudinit.readthedocs.io/en/latest/index.html).
```

Comment:
I know that Teodora does not like this pattern :P

You can sneak these links into the body of the article with _See more_.